lighthouse-core/report/report-generator.js

```diff
@@ -107,6 +99,20 @@ class ReportGenerator {
   }
 
   /**
+   * Generated time of report
+   * @return {string}
+   */
+  _generatedTime() {
+    const options = {
```

Comment:
It's called just twice (typically); not a big deal. 
Let's leave this alone for now, especially this PR doesn't change what was already there. :)